pyiron_base/jobs/master/serial.py

```diff
@@ -150,13 +150,7 @@ def start_job(self):
         Returns:
             GenericJob: start job
         """
-        if self._start_job is not None:
-            return self._start_job
-        elif len(self) > 0:
-            self._start_job = self[-1]
```

Comment:
`self[-1]` is not the same as `self[0]`

Reply:
I understand that they don't have the same effect, but I don't see why we should have different definitions for `SerialMaster` and `ParallelMaster` here